service_mesh/v2x/ossm-security.adoc

```diff
@@ -7,6 +7,20 @@ toc::[]
 
 If your service mesh application is constructed with a complex array of microservices, you can use {ProductName} to customize the security of the communication between those services. The infrastructure of {product-title} along with the traffic management features of {ProductShortName} can help you manage the complexity of your applications and provide service and identity security for microservices.
 
+.Before you begin
+
+If you have a project, add your project to the xref:../../service_mesh/v2x/installing-ossm.adoc#ossm-member-roll-modify_installing-ossm[`ServiceMeshMemberRoll` resource]. 
+
+[NOTE]
+====
+After you add the the namespace to the `ServiceMeshMemberRoll`, access to services or pods in that namespace will not be accessible to callers outside the mesh.
```

Comment:
Should we say that this is by design?  Or something like, "After you add the the namespace to the `ServiceMeshMemberRoll`, services or pods in that namespace have access to other services and pods within the mesh, but will not be accessible to callers outside the mesh unless you configure it."

Reply:
I'm not sure adding by design adds to this note. I think the note acknowledges that it's by design and I think it's implied that you can configure it differently later. I think it points out that during this step, you'll see changes.